{"name": "登录异常，测试数据是:{'accounts': '', 'pwd': 'yaoyao', 'exp': '登录账号有误'}", "status": "passed", "description": "\n        对登录接口进行异常登录的测试\n        :param args: 参数化的数据\n        :return:\n        ", "attachments": [{"name": "log", "source": "c698b8c5-0609-41b4-9108-1090dec0a661-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'accounts': '', 'pwd': 'yaoyao', 'exp': '登录账号有误'}"}], "start": 1604736646063, "stop": 1604736646176, "uuid": "9cb1850f-de67-4849-9e6f-a40e19ac7294", "historyId": "ff1d27090ef74452512974d50f385b0f", "testCaseId": "acfd50c72596b11b4392c40d7652bccf", "fullName": "script.test_login.TestLogin#test_login_error", "labels": [{"name": "story", "value": "异常数据的登录接口测试"}, {"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "baisdeMacBook-Pro.local"}, {"name": "thread", "value": "89922-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}]}